main.js

```diff
@@ -0,0 +1,54 @@
+ const electron = require("electron");
+// Module to control application life.
+ const {app} = electron;
```

Comment:
It means destructive assignment. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

I think that in this case `const app = electron;` would also work